pom.xml

```diff
@@ -72,65 +72,126 @@
         <tag>${scmTag}</tag>
```

Comment:
Above I see that you are inhering from `plugin-pom` yet using `jar` packaging, which makes no sense. If you are building a Jenkins component, use just `org.jenkins-ci:pom`.

Reply:
Whoops, I started out using a traditional maven project pom then switched in the middle to a jenkins plugin pom

Reply:
switched to `org.jenkins-ci:pom` in ef79fc334efeb8b4db5926e0954aba524bff2b53